spec/requests/internal/profile_fields_spec.rb

```diff
@@ -0,0 +1,95 @@
+require "rails_helper"
+
+RSpec.describe "/internal/profile_fields", type: :request do
+  include ActiveJob::TestHelper
+  let(:user) { create(:user) }
+
+    before do
```

Comment:
**nitpick (non-blocking):** OCD triggered again. This `before` block seems wrongly indented.